polyfill/lib/ecmascript.mjs

```diff
@@ -1452,8 +1452,7 @@ export function ToTemporalMonthDay(item, options) {
     if (calendar !== 'iso8601') {
       throw new Error(`assertion failed: missing year with non-"iso8601" calendar identifier ${calendar}`);
     }
-    RejectISODate(1972, month, day);
-    return CreateTemporalMonthDay(month, day, calendar);
+    return CreateTemporalMonthDay(month, day, calendar, 1972);
```

Comment:
Should 1972 be a named constant instead of a number literal?  Might be clearer that way.